src/resources/2023/BurningLeaves.ts

```diff
@@ -0,0 +1,65 @@
+import { Item, Monster, itemAmount, visitUrl } from "kolmafia";
+import { haveInCampground } from "../../lib";
+import { $item, $monster } from "../../template-string";
+import { get } from "../../property";
+
+const item = $item`A Guide to Burning Leaves`;
+
+export const specialLeaves: Map<Item | Monster, number> = new Map([
+  [$monster`flaming leaflet`, 11],
+  [$item`autumnic bomb`, 37],
+  [$item`impromptu torch`, 42],
+  [$item`flaming fig leaf`, 43],
+  [$item`smoldering drape`, 44],
+  [$item`distilled resin`, 50],
+  [$item`autumnal aegis`, 66],
+  [$item`lit leaf lasso`, 69],
+  [$item`forest canopy bed`, 74],
+  [$item`autumnic balm`, 99],
+  [$monster`flaming monstera`, 111],
+  [$item`day shortener`, 222],
+  [$monster`leaviathan`, 666],
+  [$item`coping juice`, 1111],
+]);
+
+/**
+ * @returns Whether or not we currently `have` the GuidetoBurningLeaves
+ */
+export function have(): boolean {
+  return haveInCampground(item);
+}
+
+/**
+ * @returns The number of leaves we have remaining
+ */
+export function numberOfLeaves(): number {
+  return itemAmount($item`inflammable leaf`);
+}
+
```

Comment:
```suggestion

function visitLeaves() {
  visitUrl("campground.php?preaction=leaves");
}

```
Need to prepend something like this each time before querying choice.php. Otherwise you won't be in the choice adventure. Used like:
```ts
visitLeaves();
visitUrl(`choice.php?pwd&whichchoice=1510&leaves=${leaves}`);
```